charts/pega/values.yaml

```diff
@@ -110,11 +110,29 @@ global:
         # Inactivity time after which requestor is passivated
         passivationTimeSec: 900
 
+
       service:
         # For help configuring the service block, see the Helm chart documentation
         # https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/README.md#service
         port: 80
         targetPort: 8080
+        tlsport: 443
+        tlstargetPort: 8443
+
```

Comment:
Not sure  about the placement of above two props? It appears, `tlsport` & `tlstargetport` are used only if `tls.enalbed` true,  but these are at two different places

Reply:
done